webapp-django/crashstats/topcrashers/jinja2/topcrashers/topcrashers.html

```diff
@@ -164,8 +164,29 @@
                                 <input type="hidden" class='ajax-signature' name="ajax-signature-1" value="{{ crash.signature }}" />
                             </div>
                             <div class="signature-icons">
+                                {% if crash.startup_count > 0 %}
+                                {% if crash.startup_count == crash.count %}
```

Comment:
I'm not sure how likely it is and it's not a big deal since we'd show the yellow flag anyway. Perhaps we can reconsider after this has been on prod for a while, if it happens often in the real world, we can change the condition.